definitions/npm/redux-thunk_v2.x.x/flow_v0.83.x-/redux-thunk_v2.x.x.js

```diff
@@ -0,0 +1,26 @@
+declare module 'redux-thunk' {
+  // Types from redux with generics removed to be compatible
+  // with redux & redux-mock-store,
+  // should use imports in the not too distant future once we
```

Comment:
We should favor the `@@` "solution" for cross libs, not the fanciest but at least we have a common thing to rely on.

So here 
```
declare module '@@redux' { ... }

declare module 'redux-thunk' {
  import type { Dispatch } from '@@redux'
}
```

Reply:
That works I'll make the change. I only wonder is there docs around this to help others in future and also, given that redux is a core lib for many, would multiple `@@redux` eventually cause conflict? For example I use both redux-saga and redux-thunk

Reply:
Your right, I haven't tested but it's probably an issue if we have several diverging `@@redux` spread in libdefs... :face_with_head_bandage:

Reply:
Should I rename this to `'@@redux-redux-thunk'`? 🤷